docker/ci/alpine/Dockerfile

```diff
@@ -27,10 +27,19 @@ RUN [ -x "$(command -v shellcheck)" ] \
     && [ -x "$(command -v helm)" ] \
     && [ -x "$(command -v aws)" ] \
     && [ -x "$(command -v z3)" ] \
+    && [ -x "$(command -v tidy)" ] \
```

Comment:
add the tools I can about and the ones that should have been add for other language support